examples/ExternalUI/Makefile

```diff
@@ -23,7 +23,9 @@ FILES_UI  = \
 
 include ../../Makefile.plugins.mk
 
+ifeq ($(LINUX),true)
```

Comment:
This check is wrong, there are other systems other than Linux that need `-ldl` like MacOS and Hurd.
So we need a blacklist rather than whitelist.
At least WINDOWS and HAIKU do not use `-ldl`, I think (but not sure) it is the same for BSD

Reply:
It'll be fine to remove this change for now.
The CI attempted a build at first of Windows ExternalUI, because of the naming error in the Makefile.